pom.xml

```diff
@@ -207,6 +207,10 @@
         <galleon.log.time>true</galleon.log.time>
         <galleon.offline>true</galleon.offline>
 
+        <!-- Operator -->
+        <kubernetes-client.version>5.12-SNAPSHOT</kubernetes-client.version>
```

Comment:
for the moment we need to compile locally `kubernetes-client` but current `master` works 🙂